src/components.ts

```diff
@@ -102,6 +103,7 @@ export async function initComponents(): Promise<AppComponents> {
 
   const fetcher = createFetchComponent()
   const memoryCache = createInMemoryCacheComponent()
+  const schemaValidator = createSchemaValidatorComponent()
```

Comment:
I'm not sure how the explorer communicates with our systems, but, the `schemaValidator` as default, enforces the `Content-type` header to be `application/json`. Would you mind either checking with the explorer team if they're sending that header or [setting the `enforceJsonContentType` to false](https://github.com/decentraland/core-components/blob/main/components/schema-validator/src/component.ts#L8) to prevent any breaking changes?